src/ducks/store.jsx

```diff
@@ -0,0 +1,18 @@
+import { legacy_createStore, applyMiddleware } from "redux";
+import localStorageMiddleware from "../middleware/localStorageMiddleware";
+import { composeWithDevTools } from "redux-devtools-extension";
+import reducers from ".";
+
+const getInitialStore = () => {
+    return {
+        testState: {}
+    }
+}
+
+export const store = legacy_createStore(
+    reducers,
+    getInitialStore(),
+    composeWithDevTools(applyMiddleware(localStorageMiddleware))
```

Comment:
I will add it to story description, lets also support thunk middleware in this pr